Fix Type, `prusalicer` => `prusaslicer`

lightmaster opened the pull request:
When trying to create some custom CSS for PrusaSliceThumbnails, I found this typo in the state thumbnail js code. I cannot find the typo in any other files using `grep`, and this bit of code does not seem to be referenced from any other part of the plugin, at least not in a way that correcting the typo breaks anything on my install.

jneilliii commented:
Thanks @lightmaster

jneilliii commented:
merged into rc and released 0.1.5rc1 version with these changes if you want to switch your release channel in the software update settings for the plugin and test.

lightmaster commented:
I removed my manual changes, switched channels and updated plugin. Looks like it took my custom CSS using the `prusa*s*licer_state_thumbnail` ID now, and I don't see any issues without the custom CSS either.
